build.gradle.kts

```diff
@@ -8,21 +8,28 @@ plugins {
     kotlin("jvm") version "1.9.22"
     kotlin("plugin.spring") version "1.9.22"
     id("net.thebugmc.gradle.sonatype-central-portal-publisher") version "1.2.3"
+    id("maven-publish")
 }
 
 group = "com.valensas"
-version = "0.2.0"
+version = "0.2.1"
 java.sourceCompatibility = JavaVersion.VERSION_17
 
 repositories {
     mavenCentral()
+    mavenLocal()
 }
 
 tasks.getByName<Jar>("jar") {
     archiveClassifier.set("")
 }
 
 dependencies {
+
+    implementation("org.springframework.cloud:spring-cloud-starter-openfeign:4.1.1")
```

Comment:
Kafka kullanmak için bu kütüphaneyi eklediğimde OpenFeign gelmemesi lazım, gereksiz bir dependency.